crates/core/tedge_api/src/mqtt_topics.rs

```diff
@@ -681,6 +689,7 @@ impl Display for Channel {
             Channel::Command { operation, cmd_id } => write!(f, "cmd/{operation}/{cmd_id}"),
             Channel::CommandMetadata { operation } => write!(f, "cmd/{operation}"),
             Channel::Health => write!(f, "status/health"),
+            Channel::Signal { signal_type } => write!(f, "sig/{signal_type}/check"),
```

Comment:
Seeing this line, I have the same concern as @albinsuresh in https://github.com/thin-edge/thin-edge.io/pull/3761#discussion_r2306382261 : why a third level?

Can we do something else the triggering a signal?

```suggestion
            Channel::Signal { signal_type } => write!(f, "sig/{signal_type}"),
```